plugins/hls-eval-plugin/README.md

```diff
@@ -274,6 +279,8 @@ To display it properly, we can exploit the fact that the output of an error is d
 ]
 ```
 
+This assumes you did not turn on exception marking (see #)
```

Comment:
It looks like a link is missing?

Reply:
Thanks, I was about to look up how to link sections and got distracted. :sweat_smile: 

Fixed :heavy_check_mark: